github/issues_test.go

```diff
@@ -149,7 +150,7 @@ func TestIssuesService_Get(t *testing.T) {
 		fmt.Fprint(w, `{"number":1, "labels": [{"url": "u", "name": "n", "color": "c"}]}`)
 	})
 
-	issue, _, err := client.Issues.Get("o", "r", 1)
+	issue, _, err := client.Issues.Get(context.Background(), "o", "r", 1)
```

Comment:
I personally liked `context.Background()` everywhere (even in tests), but it was mostly because I wasn't sure if reusing the same context value in multiple places was okay. It [turns out it is](https://twitter.com/shurcooL/status/799378333590949893) (/cc @rakyll), because of how it's [implemented](https://gotools.org/context#Background).

I still kinda like/don't mind `context.Background()` everywhere because it's readable and I don't think saving a bit of typing is worth it, but I'm also okay with with `var bg = context.Background()` or similar (in tests only).

Previous discussions:

- https://github.com/tambet/go-asana/pull/4#discussion_r80197313
- https://twitter.com/rogpeppe/status/798970167338422272 (/cc @rakyll)